Create Anagrams.py

devkapilbansal opened the pull request:
This is the solution of hacker earth anagrams problem that was given

devkapilbansal committed:
Create Anagrams.py

This is the solution of hacker earth anagrams problem that was given

pandafy commented:
Can you add question in comments ?

ekta1999 commented:
Yes, It's mandatory


On Thu, Oct 3, 2019 at 5:33 PM TheOneAboveAllTitan <notifications@github.com>
wrote:

> Can you add question in comments ?
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/C0deAttackC0mmunity/Programming-question/pull/1?email_source=notifications&email_token=AKFJVHQQ5TOEJ3IK6HH35HTQMXNQFA5CNFSM4I5BVYF2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEAH7BVQ#issuecomment-537915606>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AKFJVHXWNR4FBDRUUYPNKC3QMXNQFANCNFSM4I5BVYFQ>
> .
>

codeattackcommunity commented:
@KapilBansal320 Add question description in comments or question link at the starting in comments then we'll review it again and if find valid then accept.

devkapilbansal committed:
Update Anagrams.py

https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/anagrams-651/
